frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -23,144 +24,227 @@ export type MatchesListProps = {
       setExpanded(isExpanded ? panel : false);
    };
 
    function reloadAllList() {
-      MatchesListRestApi()
-         .getMatchesList(tournamentId)
-         .then((matchesListDto) => {
-            const newMatchesListItems = returnMatchListItemsFromMatchesListDto(matchesListDto, reloadAllList);
-            setMatchesListItems(newMatchesListItems)
-         });
+      TournamentMatchesListRestAPI()
+         .getTournamentTeamsList(tournamentId)
+         .then((tournamentMatchesListDto) => setTournamentMatchesListDto(tournamentMatchesListDto));
    }
 
    return (
       <StyledMatchesList>
-         {matchesListItems ? matchesListItems.map((item, index) => returnMatchItem(item, index, expanded, handleChangeExpander)) : "Oczekiwanie na pobranie..."}
+         {matchesList
+            ? matchesList
+               .map((item, index) => returnMatchItem(item, index, expanded, handleChangeExpander))
+            : "Oczekiwanie na pobranie..."}
       </StyledMatchesList>
    )
 };
 
-const returnMatchItem = (matchItemBase: MatchListItem, index: number, expanded: string | boolean, handleChangeExpander: (panel: string | boolean) => (event: any, isExpanded: string | boolean) => void) => (
+const returnMatchItem = (
+   {level, matchId, matchNumber, matchStatus, onClickTeam, tableNumber, team1, team2, winnerId}: MatchItemType,
+   index: number,
+   expanded: string | boolean,
+   handleChangeExpander: (panel: string | boolean) => (event: any, isExpanded: string | boolean) => void
+) => (
    <MatchItem
-      level={matchItemBase.level}
-      matchNumber={matchItemBase.matchNumber}
-      matchId={matchItemBase.matchId}
-      matchStatus={matchItemBase.matchStatus}
-      onClickTeam={matchItemBase.onClickTeam}
-      team1={matchItemBase.team1}
-      team2={matchItemBase.team2}
-      winnerTeamId={matchItemBase.winnerId}
-      key={matchItemBase.matchNumber}
+      key={matchNumber}
+      level={level}
+      matchNumber={matchNumber}
+      matchId={matchId}
+      matchStatus={matchStatus}
+      onClickTeam={onClickTeam}
+      tableNumber={tableNumber}
+      winnerTeamId={winnerId}
+      team1={team1}
+      team2={team2}
       expanded={expanded}
       handleChangeExpander={handleChangeExpander}
    />
 )
 
-const returnMatchListItemsFromMatchesListDto = (matchesListDto: MatchesListDto, reloadAllList: () => void): MatchListItem[] => {
-   return matchesListDto.queue.map((matchesItem) => {
-
-      function findStatus(): MatchStatus {
-         if (matchesItem.status === "started") return MatchStatus.STARTED;
-         if (matchesItem.status === "ended") return MatchStatus.FINISHED;
-         if (matchesItem.status === "enqueued") return MatchStatus.NO_TABLE;
-         if (matchesItem.status === "noTeams") {
-            if (matchesItem.team1Id || matchesItem.team1Id) return MatchStatus.NO_ONE_TEAM;
-            return MatchStatus.NO_TEAMS;
+const returnMatchList = (
```

Comment:
Raczej nie piszemy return w nazwach funkcji - wiadomo przezciez, ze return skoro bedziesz mial nazwe np. `matchList`.

Reply:
zapomniałem poprawić! Dzięki!

Reply:
Tylko mam zmienne o tych samych nazwach! ;\
Wrzucę to na razie tak - bo deadline - i poprawię w przyszłości. Robię screen